### PR TITLE
docs: prepare v18.0

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -4,13 +4,6 @@ order: 1
 sidebar_position: 1
 ---
 
-:::tip
```
Comment (mpoke) [inline docs/docs/index.md:7]: @MSalopek is there something else that needs to be done for the docs?

File: UPGRADING.md
Comment (mpoke) [inline UPGRADING.md:1]: @MSalopek could you please review?

File: RELEASE_NOTES.md
Comment (mpoke) [inline RELEASE_NOTES.md:1]: @Btruax could you please review?

File: docs/docs/index.md (hunk 1 above)
Comment (MSalopek) [reply docs/docs/index.md:7]: Cosmwasm and build section updates. Those can be main only

**Review by MSalopek — APPROVED** [review]

**Pull request merged**